CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ You can find a list of previous releases on the [github releases](https://github
 
 ### Changed
 - Bump CLI version to v0.18.3 (#3959)
+- Server will deep merge configuration files when loading. No need to copy the whole config section, specify only those fields that needs overriding. (#4165)
```

Comment:
Maybe update this in latest release section

Reply:
Rebased and updated